src/nurse/management/commands/_notifications.py

```diff
@@ -0,0 +1,30 @@
+from django.conf import settings
+from onesignal_sdk.client import Client
+
+app_id = settings.ONESIGNAL_APP_ID
+api_key = settings.ONESIGNAL_API_KEY
+
+if not app_id or not api_key:
+    raise ValueError(
+        "ONESIGNAL_APP_ID or API_KEY is missing from the environment variables."
+    )
```

Comment:
The message is off as the environment variable name changed.
Wording is also off it's not "or" but "and".
Try with the Walrus approach https://github.com/mynotif/mynotif-backend/pull/121#discussion_r1292296027